assets/models.py

```diff
@@ -3,19 +3,20 @@
 
 
 class Asset(BaseModel):
-    name = models.CharField(max_length=50)
+    asset_name = models.CharField(max_length=50)
```

Comment:
"title" could be a better option instead of a name. 
inheriting "TitleDescriptionModel" from django.extensions could be a better option.